Add backgroundContainerError and textError tokens

paulasuarezb opened the pull request:
#1708 Rowlist danger

paulasuarezb committed:
#1708 Rowlist danger

yceballost commented on tokens/blau.json:
```diff
@@ -25,6 +25,11 @@
       "description": "white"
     },
+    "backgroundContainerError": {
+      "value": "{palette.white}",
```
sorry, I can't understand why `white` is set up for this token for all brands. Shouldn't it be equivalent to `errorLow`?

![image](https://github.com/Telefonica/mistica-design/assets/6722153/6aae4276-c580-4837-a965-76496bef3a4f)

This comment apply to all skins

paulasuarezb replied:
should be the same color in backgroundContainerError and errorLow, you're right! I'll change it

yceballost replied:
You didn't change it! xD

I think you make changes in dark mode and not in light mode

yceballost commented:
Please, change the PR name to something more specific. Like "Add backgroundContainerError and textError tokens"

aweell reviewed (commented):
I'm missing `textErrorInverse` token

paulasuarezb committed:
tokens changes

added textErrorInverse token and changed backgroundContainerError palette same as errorLow

aweell commented on tokens/movistar.json:
```diff
@@ -1093,6 +1113,16 @@
+      "type": "color",
+      "description": "pepper"
+    },
+    "textErrorInverse": {
```
`textErrorInverse` in dark should be = `textError` in all brands

paulasuarezb committed:
textErrorInverse dark = textError

textErrorInverse in dark should be = textError in all brands

yceballost commented on tokens/blau.json:
```diff
@@ -668,6 +683,11 @@
       "description": "darkModeGrey"
     },
+    "backgroundContainerError": {
+      "value": "{palette.blauRed10}",
```
```suggestion
      "value": "{palette.darkModeGrey}",
```
Here the right color should be the same as `backgroundContainer` in dark mode, i.e. darkModeGrey

aweell replied:
+1

* All `backgroundContainerError` tokens in **light** should have the same value that `errorLow`
* All `backgroundContainerError` tokens in **dark** should have the same value that `backgroundContainer`

aweell commented:
You can check the effect of your changes here: https://mistica-design.vercel.app/tokens-map?branch=%231708-Rowlist-danger&skin=movistar&tokenType=color&activeColor=undefined

paulasuarezb committed:
update telefonica

paulasuarezb committed:
backgroundContainerError fix

paulasuarezb committed:
mov fix

paulasuarezb commented:
I think I got it now 😅 that url was really helpful

aweell commented on tokens/o2-new.json:
```diff
@@ -482,6 +487,16 @@
       "description": "o2Red65"
     },
+    "textError": {
+      "value": "{palette.o2Red45}",
```
```suggestion
      "value": "{palette.o2Red65}",
```

aweell commented on tokens/o2-new.json:
```diff
@@ -482,6 +487,16 @@
+    "textError": {
+      "value": "{palette.o2Red45}",
+      "type": "color",
+      "description": "o2Red45"
```
```suggestion
      "description": "o2Red65"
```

aweell commented on tokens/vivo-new.json:
```diff
@@ -450,6 +455,16 @@
+      "description": "pepper"
+    },
+    "textErrorInverse": {
+      "value": "{palette.pepper}",
```
```suggestion
      "value": "{palette.white}",
```

yceballost replied:
why white? in dark mode we can use red in inverse 🤔

aweell replied:
This is light mode in dark mode is pepper which is correct

aweell commented on tokens/vivo-new.json:
```diff
@@ -450,6 +455,16 @@
+    "textErrorInverse": {
+      "value": "{palette.pepper}",
+      "type": "color",
+      "description": "pepper"
```
```suggestion
      "description": "white"
```

paulasuarezb committed:
text error tokens

changed O2-new text-Error and vivo-new textErrorInverse

paulasuarezb committed:
Update tokens/vivo-new.json

Co-authored-by: Alex Bueno <alejandro.rodriguezbueno@telefonica.com>
